KeeTrayTOTP/Libraries/QRCoder/QRCodeGenerator.cs

```diff
@@ -569,6 +569,47 @@ private static bool IsBlocked(Rectangle r1, List<Rectangle> blockedModules)
 
             private static class MaskPattern
             {
+                // NOTE: Methods are invoked via reflection (typeof(MaskPattern).GetMethods()))
```

Comment:
Didn't do much with the qr code stuff. Do you know why they're using reflection for this?

Reply:
Unfortunately not. The files were copied over from a nuget package (https://github.com/codebude/QRCoder) when the QR Dialog was added. When cleaning up I did not notice that reflection was used. The irony is that in the meantime that package moved away from reflection as well (https://github.com/codebude/QRCoder/blob/master/QRCoder/QRCodeGenerator.cs), so I could update and clean up as well. 

I'm struggling a bit with how to handle dependencies in the plgx scenario. I'd rather add a reference to the package but I've not found a way to get this working.

Reply:
As a next step we could grab the latest version of the files from the QRCode repository. Will add that as an item on the board.